prow/knative/config.yaml

```diff
@@ -172,6 +172,14 @@ tide:
     missingLabels: *knative_tide_missing_labels
     author: knative-prow-robot
     reviewApprovedRequired: false
+  - repos:
+    # Allow PRs for test-infra to be automatically merged without manual approval
+    - "knative/test-infra"
+    labels:
+      - "cla: yes"
```

Comment:
How about adding a label like https://github.com/kubernetes/test-infra/blob/7d5e552c49619d424c4cb20429c8ec0ca1bba750/config/prow/config.yaml#L551, to make this better controllable?

Reply:
Because that will allow you to do something like https://github.com/knative/test-infra/pull/2729#issuecomment-816998924

Reply:
Good point. Updated in https://github.com/knative/test-infra/pull/2746